torch/csrc/jit/ir.h

```diff
@@ -111,7 +111,8 @@ struct Node : public Attributes<Node> {
   Graph* graph_;
   size_t unique_ = 0;          // unique id
   size_t stage_ = 0;           // 0-forward, 1-backward, 2-double-backward,...
-  std::string debug_name_;
+  std::string pretty_name_;
+  std::string overriding_unique_name_;
```

Comment:
How about just `name_` and `unique_name_`?

Reply:
about `name_`: see my reply above
about `unique_name_`: since there's already a function called `uniqueName()` which does something more complicated than `return unique_name_`, I preferred to have a name that implies that something more complicated is going on (because it is).

Happy to change either/both though if you prefer the shorter names.

Reply:
I thought to have `unique_name_` make `uniqueName()` return just this if it's set, and do the more complicated string dance otherwise